File: MODUL-1/Python/PRAK107-2310817220002-RaudatulSholehah.py
print(f"Diketahui : ")
a = int(input("Variabel a bernilai :"))
b = int(input("Variabel b bernilai :"))
c = int(input("Variabel c bernilai :"))

Harga_tanah_Per_Meter = 85000
hasilkeliling = a + b + c
totalhasil = hasilkeliling * Harga_tanah_Per_Meter

print(f"Panjang sisi segitiga berturut-turut adalah {a, b, c}")
print(f"Keliling Tanah Pak Dengklek adalah {hasilkeliling}")
print(f"Harga tanah Per Meter adalah {Harga_tanah_Per_Meter}")
print(f"Jawaban : ")
print(f"Biaya yang diperlukan Pak Dengklek adalah Rp {totalhasil}")

File: MODUL-1/C/PRAK107-2310817220002-RaudatulSholehah.c
#include <stdio.h>
int main()
{
    int a, b, c;

    printf("Diketahui : \n");
    printf("Inputkan nilai sisi segitiga a: ");
    scanf("%d", &a);

    printf("Inputkan nilai sisi segitiga b: ");
    scanf("%d", &b);

    printf("Inputkan nilai sisi segitiga c: ");
    scanf("%d", &c);

    int Harga_tanah_Per_Meter = 85000;
    int hasilkeliling = a + b + c;
    int totalhasil = hasilkeliling * Harga_tanah_Per_Meter;

    printf("Panjang sisi segitiga berturut-turut adalah %d, %d, %d\n", a, b, c);
    printf("Keliling Tanah Pak Dengklek adalah %d\n", hasilkeliling);
    printf("Harga tanah Per Meter adalah %d\n", Harga_tanah_Per_Meter);
    printf("Jawaban : \n");
    printf("Biaya yang diperlukan Pak Dengklek adalah : Rp %d\n", totalhasil);

    return 0;
}
